ripley/api/lti_controller.py

```diff
@@ -108,16 +178,51 @@ def initiate_login():
         raise InternalServerError({'message': str(e)})
 
 
+@app.route('/api/lti/add_user', methods=['GET', 'POST'])
+def launch_add_user():
+    return _launch_tool('add_user')
+
+
+@app.route('/api/lti/course_captures', methods=['GET', 'POST'])
+def launch_course_captures():
+    return _launch_tool('course_captures')
```

Comment:
@lyttam - Afaik, we're not supporting legacy course capture. Maybe we should verify with Miles.

Reply:
Confirmed: Course Captures is cashing in its pension and retiring.

Reply:
Thanks - amended.